### PR TITLE
Improves style for shell examples

### PR DESCRIPTION
**Pull request by gguuss**
Addresses comments missed in PR for adding GCS object support.

### CODE REVIEW

File: language/analysis/README.md [hunk 1]
```diff
@@ -32,24 +32,56 @@ mvn clean compile assembly:single
 We can then run the assembled JAR file with the `java` command. The variable $COMMAND takes
 three values `entities`, `sentiment`, or `syntax`.
 
+Basic usage:
+
 ```
```
Comment (lesv) [inline language/analysis/README.md]: You can add bash after the "`"'s

**Comment by tswast** [comment]
+1 to making them a real sh script. This would allow shellcheck to check it for errors.

You can use https://github.com/campoy/embedmd to include it in the README.

File: language/analysis/README.md (hunk 1 above)
Comment (gguuss) [inline language/analysis/README.md]: I will add a .BAT / .SH script in a separate PR and will update README after.

**Comment by gguuss** [comment]
I've [added an issue](https://github.com/GoogleCloudPlatform/java-docs-samples/issues/437) to separate shell code into scripts and optionally embed the script sources into the readme.md.

**Pull request merged**